### PR TITLE
feat(extension-api): adding the id property to the WebviewPanel object

### PR DESCRIPTION
**Pull request by axel7083**
### What does this PR do?

To use the `navigation.navigateToWebview` we need the `id` of the webview, but it is not on the WebviewPanel object...
 The only way to get this object is to list the webviews and search by title `windows.listWebviews().find(panel => panel.title === "<my-title>"`.

### What issues does this PR fix or reference?

N/A

### How to test this PR?

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

**Review by benoitf — CHANGES_REQUESTED** [review]
edited:
we have internalId so it means we want to keep it secret/hidden, if we need exposure, need to rename

**Comment by benoitf** [comment]
to comment to myself, if `id` needs to be a public thing, then the variable name should not be 'internalId' that we expose at the end.

File: packages/main/src/plugin/webview/webview-panel-impl.ts [hunk 1]
```diff
@@ -76,6 +76,10 @@ export class WebviewPanelImpl implements WebviewPanel {
     this.#viewType = panelDetails.viewType;
   }
 
+  get id(): string {
```
Comment (benoitf) [inline packages/main/src/plugin/webview/webview-panel-impl.ts:79]: if id needs exposure then probably to rename `#internalId` to `#id`

and just rename existing
```
get internalId(): string {
```

to 
```
get id(): string {
  this.#id
}
```

Comment (axel7083) [reply packages/main/src/plugin/webview/webview-panel-impl.ts:79]: Good idea

**Review by benoitf — APPROVED** [review]

**Comment by benoitf** [comment]
on hold, maybe switch to the usage of `webview.reveal()` method

**Comment by axel7083** [comment]
> on hold, maybe switch to the usage of `webview.reveal()` method

Just tested, the method is working well, closing

**Pull request closed without merging**